robottelo/utils/__init__.py

```diff
@@ -80,3 +91,178 @@ def slugify_component(string, keep_hyphens=True):
     if not keep_hyphens:
         string = string.replace('-', '_')
     return re.sub("[^-_a-zA-Z0-9]", "", string.lower())
+
+
+# Manifest Cloning
+class ManifestCloner:
+    """Manifest clonning utility class."""
```

Comment:
```suggestion
    """Manifest cloning utility class."""
```

Reply:
@adarshdubey-star Updated in the https://github.com/SatelliteQE/robottelo/pull/10232 as there was too much dependencies in both the PRs. Can you please review the changes there.